### PR TITLE
Improve feedback UI

### PR DESCRIPTION
**Pull request by pradh**
Includes the following changes:

* Ranking tile and top-events tile includes thumbs-down
   * TODO: the top-events tile might need some css tweaks
* When LOG_QUERY is False, no query-level thumbs-down shown (chart-level already doens't show)
* With `enable_demo=True` url param, no thumbs-down shown

Screenshots:

image

image

### CODE REVIEW

File: static/js/components/nl_feedback.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import React, { useContext, useState } from "react";
+
+import { NlSessionContext } from "../shared/context";
+import {
+  CHART_FEEDBACK_SENTIMENT,
+  getNlChartId,
+} from "../utils/nl_interface_utils";
+
+interface NLChartFeedbackPropType {
+  id: string;
```
Comment (shifucun) [inline static/js/components/nl_feedback.tsx:31]: comment this is a fix format string

File: static/js/components/nl_feedback.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+export function NLChartFeedback(props: NLChartFeedbackPropType): JSX.Element {
```
Comment (shifucun) [inline static/js/components/nl_feedback.tsx:34]: NlChartFeedback for consistency?

**Review by shifucun — APPROVED** [review]

File: static/js/components/nl_feedback.tsx (hunk 1 above)
Comment (pradh) [reply static/js/components/nl_feedback.tsx:33]: Done.

File: static/js/components/nl_feedback.tsx (hunk 2 above)
Comment (pradh) [reply static/js/components/nl_feedback.tsx]: Done.

**Review by pradh — COMMENTED** [review]
Thanks.

**Pull request merged**